hw/ip/usbdev/doc/_index.md

```diff
@@ -111,11 +120,11 @@ The following table summarizes how the different input signals relate to the USB
 
 The USB device features the following non-data pins.
 
-|  External Pins | Internal Signals | Notes |
-|----------------|------------------|-------|
-| sense (VBUS)   | sense_i          | The sense pin indicates the presence of VBUS from the USB host. |
-| [pullup]       | pullup_o         | When the pullup_o asserts a 1.5k pullup resistor should be connected to D+. This can be done inside the chip or with an external pin. A permanently connected resistor can also be used. |
-| [suspend]      | suspend_o        | The suspend pin indicates to the USB transceiver that a constant idle has been detected on the link and the device is in the Suspend state (see Section 7.1.7.6 of the [USB 2.0 specification](https://www.usb.org/document-library/usb-20-specification)). |
+|  External Pins | Internal Signals         | Notes |
+|----------------|--------------------------|-------|
+| sense (VBUS)   | sense_i                  | The sense pin indicates the presence of VBUS from the USB host. |
+| [pullup]       | dp_pullup_o, dn_pullup_o | When dp_pullup_o or dn_pullup_o asserts a 1.5k pullup resistor should be connected to D+ or D-, respectively. This can be done inside the chip or with an external pin. A permanently connected resistor can also be used. Only one of the pullup signals can be asserted at any time. The selection is based on the `pinflip` bit in {{< regref "phy_config" >}}. By default, dp_pullup_o, i.e., the pullup on D+ is used. |
```

Comment:
"A permanently connected resistor could be used if the pin flip feature is not needed, but this is not recommended because there is then no way to force the device to appear to unplug."

"By default ... used" -> "Because this is a Full Speed device the resistor must be on the D+ pin, so when pinflip is zero the dp_pullup_o is used."

Reply:
Done.